conversion/conversion.go

```diff
@@ -156,6 +159,11 @@ func getDbNameFromSQLConnectionStr(driver, sqlConnectionStr string) string {
 	case constants.SQLSERVER:
 		splts := strings.Split(sqlConnectionStr, "?database=")
 		return splts[len(splts)-1]
+	case constants.ORACLE:
+		// connection string formate : "oracle://user:password@104.108.154.85:1521/XE"
+		substr := sqlConnectionStr[9:]
+		dbName := strings.Split(substr, ":")[0]
```

Comment:
This seems to return user instead of db name, I take it the dbname in the sample string is XE?

Reply:
Actually, XE is the SID. Here user needs to be considered. In oracle User / Owner is essentially the same as the Schema. In the queries we need to pass the schema.